cmd/cmd.go

```diff
@@ -65,12 +67,14 @@ func NewPackCommand(logger ConfigurableLogger) (*cobra.Command, error) {
 
 	rootCmd.AddCommand(commands.Build(logger, cfg, &packClient))
 	rootCmd.AddCommand(commands.Rebase(logger, cfg, &packClient))
-	rootCmd.AddCommand(commands.InspectImage(logger, &cfg, &packClient))
+	// TODO: is the config the base of truth for what run image mirrors are displayed??
+	//       or do we use the run image mirrors located on the image??
+	rootCmd.AddCommand(commands.InspectImage(logger, writer.NewFactory(), cfg, &packClient))
```

Comment:
When inspecting an image, I would expect the run image mirrors on the image to be the source of truth, while the config is the base for future images built with `pack`.

Reply:
Ya comment should be removed, as things stand we print run images from Both, 
- run images specified in the `config` are indicated by a `(user-provided)` field.